Example/watchOSSample/SampleWatchAppWatchKitExtension/ExtensionDelegate.swift

```diff
@@ -29,17 +30,25 @@ class ExtensionDelegate: NSObject, WKExtensionDelegate, MessagingDelegate {
       }
     }
     Messaging.messaging().delegate = self
+    let remoteConfig = RemoteConfig.remoteConfig()
+    remoteConfig.fetchAndActivate { status, error in
+      guard error == nil else {
+        print("error:" + error.debugDescription)
+        return
+      }
+      print("value:\n" + remoteConfig["test"].stringValue!)
```

Comment:
might be helpful if we also print the fetch source?

Reply:
Done. I specify it in the output that if the config is setup on console.